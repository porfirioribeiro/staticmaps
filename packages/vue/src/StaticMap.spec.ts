import { mount } from '@vue/test-utils';
import StaticMapVue from './StaticMap.vue';

test('renders', () => {
  const wrapper = mount(StaticMapVue, {
    props: {
      options: {
        width: 345,
        height: 138,
        padding: [10, 10],
        bbox: [-61.0698239073564793, -21.3991246364298995, -61.0529563571889184, -21.407334765312239],
      },
    },
  });
  expect(wrapper.html()).toMatchSnapshot();
});
