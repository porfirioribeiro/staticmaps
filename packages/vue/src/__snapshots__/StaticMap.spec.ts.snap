// Vitest Snapshot v1

exports[`renders 1`] = `
"<svg width=\\"345\\" height=\\"138\\" viewBox=\\"0 0 345 138\\">
  <image href=\\"https://tile.openstreetmap.org/14/5412/9189.png\\" x=\\"-91\\" y=\\"-75\\" width=\\"256\\" height=\\"256\\"></image>
  <image href=\\"https://tile.openstreetmap.org/14/5413/9189.png\\" x=\\"165\\" y=\\"-75\\" width=\\"256\\" height=\\"256\\"></image>
</svg>
<div>Hello</div>"
`;
